### ISSUE TITLE
[bug] `<rh-card>` use of landmarks could render a page less usable for assistive tech

### ISSUE THREAD

**Issue by nikkimk** [RedHat-UX/red-hat-design-system#1233]
### Describe the bug

`rh-card` should not use aria-landmarks since many card can exist one one page.

According to Deque University's [Semantic Structure and Navigation (WCAG 2.1): Best Practices for Landmarks](https://dequeuniversity.com/class/semantic-structure/landmarks/best-practices):

> The total number of landmarks SHOULD be minimized to the extent appropriate for the content.
>
> There is no technical limit on the number of landmarks you can implement in a page, but one of the main purposes of landmarks is to allow blind users to quickly find and navigate to the appropriate landmark, so you should keep the total number of landmarks relatively low. If you don't, screen reader users will have to sort through too much extra information to find what they're looking for.
>
> For example, just because you can mark a list of links as a navigation landmark doesn't mean that you always should. If you have, say, one to three navigation landmarks (each labeled with aria-label or aria-labelledby), users will have no problem. But if you have 15 to 20 navigation landmarks, it may take users a long time to figure out what all navigation lists are for, and which one contains the information they're looking for.

### Which Component?

rh-card

### Steps to reproduce

From a page with multiple cards, count the total number the landmarks.
Every card adds two landmarks to the total count.

### Expected behaviour

There should be fewer landmarks on a page.

### Operating System (OS)

macOS

### Browser

Chrome

### Additional Info

Unless we want guidelines limiting the number of cards on a page, I suggest we remove the header and footer landmarks from card.

**Comment by bennypowers** [comment]
@nikkimk is the solution to guide page authors to reduce use of rh-card, then?

as well, this means we should expedite discussions on rh-surface

**Comment by bennypowers** [comment]
@adamjohnson how should we proceed here?

1. never use landmarks in card -  make everything a div
2. always use landmarks (i.e. close this issue with no change)
3. make landmarks opt out (i.e. by default uses landmarks, adding an attribute makes everything a div)
4. make landmarks opt in (i.e. by default makes everything a div, adding an attribute uses landmarks)
5. some other solution

in options (3) and (4) it doesn't have to be an attribute necessarily that makes the difference, it could be special slots or some global config

**Comment by adamjohnson** [comment]
Let's go with option `#1` - never use landmarks in card - make everything a `div`.
